pr2eus/pr2-interface.l

```diff
@@ -407,7 +407,7 @@ Example: (send self :gripper :rarm :position) => 0.00"
 
 (defmethod pr2-interface
   (:angle-vector-with-constraint
-   (av1 &optional (tm 3000) (arm :arms) &key (rotation-axis t) (translation-axis t) &rest args)
+   (av1 &optional (tm 3000) (arm :arms) &key (rotation-axis t) (translation-axis t) &allow-other-keys)
```

Comment:
There is no meaning in defining `&rest` arguments after `&key` declaration, since sending anything after the keys will led to errors like `no such keyword` or `keyword expected`. Never being reached, args value will always be nil.

If the purpose of `&rest` here is to allow the user to pass undefined arguments without errors (since args is not used anywhere), the use of `&allow-other-keys` should be correct. If not, would be better to erase `&rest` altogether, avoiding misleading coding.